GVFS/GVFS.UnitTests.Windows/Windows/CommandLine/SparseVerbTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using GVFS.CommandLine;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace GVFS.UnitTests.Windows.Windows.CommandLine
+{
+    [TestFixture]
+    public class SparseVerbTests
+    {
+        // Shorter name for readability
+        private static char dirSep = Path.DirectorySeparatorChar;
+
+        [TestCase]
+        public void PathCoveredBySparseFolders_RootPaths()
+        {
+            string rootPaths = $"a.txt{SparseVerb.StatusPathSeparatorToken}b.txt{SparseVerb.StatusPathSeparatorToken}c.txt{SparseVerb.StatusPathSeparatorToken}";
+
+            ConfirmAllPathsCovered(rootPaths, new HashSet<string>());
+            ConfirmAllPathsCovered(rootPaths, new HashSet<string>(StringComparer.OrdinalIgnoreCase) { "A" });
+            ConfirmAllPathsCovered(rootPaths, new HashSet<string>(StringComparer.OrdinalIgnoreCase) { "A", $"B{dirSep}C" });

```

Comment:
Maybe have some static `HashSet`s for these?

Reply:
I will give that a try and use them where it improves readability.